restore/data.go

```diff
@@ -105,7 +104,7 @@ func restoreSingleTableData(queryContext context.Context, fpInfo *filepath.FileP
 		// If this occurs we need to error out, as subsequent COPY statements
 		// will hang indefinitely waiting to read from pipes that the helper
 		// was expected to set up
-		if backupConfig.SingleDataFile {
+		if backupConfig.SingleDataFile || resizeCluster {
```

Comment:
General question (unrelated to the patch): why do we check agents on erroring out only in case of SingleDataFile (originally)?

Reply:
I don't know and I fixed it

Reply:
And why you fixed it by extending the condition, and not by complete removal of if-clause?

Reply:
Ok, let it remains a mystery, but this moment should be investigated somewhen

Reply:
> And why you fixed it by extending the condition, and not by complete removal of if-clause?

because only in these two cases helpers are started

Reply:
reveted, I created separate task for it